modules/migration-installing-legacy-operator.adoc

```diff
@@ -18,6 +18,11 @@ ifdef::installing-mtc,installing-mtc-restricted[]
 You can install the legacy {mtc-full} Operator manually on {product-title} versions 4.2 to 4.5.
 endif::[]
 
+[NOTE]
+====
+MTC 1.7 supports migrations from {product-title} 3.11 to 4.8. MTC 1.8 only supports migrations from {product-title} 4.9 and later.
```

Comment:
This line looks good, BUT all the steps below mention 1.8 image, which should not be there.

In short, There should be no steps related to MTC 1.8 , in the OCP 3.x installation docs.

Reply:
![image](https://github.com/openshift/openshift-docs/assets/30953139/3a848a6c-67da-4f65-9541-0ac03eecf43f)

Reply:
cc @dymurray can you please confirm